### ISSUE TITLE
nChillu+Virama+Rra combination not supported in Malayalam 

### ISSUE THREAD

**Issue by kalapi** [googlefonts/googlesans#643]
### Issue information:

nChillu + Virama + Rra combination not working. (Text: `ൻ്റ`)

Image reference:
![C3MVnAvmd8x7ywH](https://github.com/user-attachments/assets/45fc4456-0a96-41f8-9e57-9585272ca095)

### Resolution:

- [x] The code is included in `malayalam.fea` on line 128 but missing a context.

    Was:
    ```
    lookup MALM_dflt_akhn_chilluConjuncts {
            sub nChillu-malayalam rra-malayalam by n_rra-malayalam;
    } MALM_dflt_akhn_chilluConjuncts;
    ```
    Should be:
    ```
    lookup MALM_dflt_akhn_chilluConjuncts {
        sub nChillu-malayalam halant-malayalam rra-malayalam by n_rra-malayalam;
    } MALM_dflt_akhn_chilluConjuncts;
    ```
- [x] The sample text in the `qa/shaping_input/malayalam_default_ortho_akhand_forms.toml` is incorrect and needs to be updated to the correct input.

    Was:
    ```
    "ന്‍റ", # Malayalam Chillu conjuncts
    ```
    
    Should be:
    ```
    "ൻ്റ", # Malayalam Chillu conjuncts
    ```

cc: @chrissimpkins @MariannaPaszkowska

**Comment by chrissimpkins** [comment]
internal bug tracker for this issue: b/354001927

**Comment by MariannaPaszkowska** [comment]
Many thanks @kalapi 

Fix in #644 
 
<img width="1166" alt="Screenshot 2024-09-05 at 2 52 47 PM" src="https://github.com/user-attachments/assets/ba8a98f9-8c1d-4681-bae1-371c326b0d83">
<img width="1162" alt="Screenshot 2024-09-05 at 2 52 54 PM" src="https://github.com/user-attachments/assets/2ec4b2ef-9d0a-4e74-bc1a-9c118529bf6e">

<img width="1157" alt="Screenshot 2024-09-05 at 4 13 15 PM" src="https://github.com/user-attachments/assets/714b0380-e396-4024-9b87-fad9cdb783d3">
<img width="1173" alt="Screenshot 2024-09-05 at 4 13 10 PM" src="https://github.com/user-attachments/assets/4c7cf6d1-cd4d-4b76-a9c6-2bcf342064c9">

<img width="794" alt="Screenshot 2024-09-05 at 4 13 34 PM" src="https://github.com/user-attachments/assets/059bb14a-4682-4f81-b17a-d5d140296385">

**Comment by chrissimpkins** [comment]
Request to base our Android build target on this fix in https://github.com/googlefonts/googlesans/pull/644#issuecomment-2332613419
